mantidproject/mantid
# Normalization with collection time for HB2AReduce

**Description of work.**



The monitor counts on HB2A are showing random fluctuation and therefore the default normalization with monitor counts may not be suitable. As requested by the HB2A team, we added in the capability in `HB2AReduce` algorithm to normalize data with collection time.

**To test:**

Launch `HB2AReduce` algorithm and input any run to reduce the data. We are expected to see the outputs of reduced data with both monitor (as before) and collection time normalization. The output file will be with the specified stem name for normalization with monitor counts, like `stem_name.gss`. The output file with collection time normalization will be named as `stem_name_norm_time.gss` in this case.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
